src/rdkafka.h

```diff
@@ -4798,6 +4805,33 @@ RD_EXPORT
 void rd_kafka_metadata_destroy(const struct rd_kafka_metadata *metadata);
 
 
+/**
+ * @brief Get the id of \p node.
+ *
+ * @param node The Node instance.
+ * @return The node id, or -1 is \p node is NULL.
+ */
+RD_EXPORT
+int rd_kafka_Node_id(rd_kafka_Node_t *node);
```

Comment:
If we don't foresee this object to be mutable then we should consider consting the node param.

Reply:
done